polymer.json

```diff
@@ -16,7 +16,7 @@
    "index.html",
    "main.py"
   ],
-  "includeDependencies": [
+  "extraDependencies": [
```

Comment:
We should probably keep `includeDependencies` so polymer-cli 0.17.0, which is the official release version, would work.